### ISSUE TITLE
Feature Request: Unnecessarily huge plotly images

### ISSUE THREAD

**Issue by iirekm** [neptune-ai/neptune-client#1870]
Neptune uses Plotly's `write_html` inside `File.as_html` or `NeptuneCallback` listener (Optuna integration). The generated files are huuuge (almost 5 MB each), even if few data in those charts, which leads to numerous problems:

- it could easily fill even 20 GB during longer training sessions; I often got "no space left on device"
- at the end of training I got ` Waiting for the remaining nnnnn operations to synchronize with Neptune. Do not kill this process.` (when nnnn were always multiple thousands)
- charts load slowly when using slower mobile internet (has to download those 5 MB for each of then)
- last but not least: neptune.ai probably has to pay more for such unnecessary storage

Why not replace `write_html(buffer)` with `image.write_html(buffer, include_plotlyjs="cdn")`? This, instead of embedding huge plotly libraries inside the HTML, downloads them from a CDN. It's not a problem since internet connection is required to use neptune.ai anyway.

### Workaround 

Until this issue is fixed, I'm using the following patch, which fixes the problems:
```python
def patch_neptune():
    def _export_plotly_figure(image):
        buffer = StringIO()
        image.write_html(buffer, include_plotlyjs="cdn")
        buffer.seek(0)
        return buffer.getvalue()
    import neptune.internal.utils.images as imgs
    imgs._export_plotly_figure = _export_plotly_figure
```

**Comment by SiddhantSadangi** [comment]
Hey @iirekm 👋 

We had discussed this internally in the past. The issue is that the same API is also used in Neptune's on-premise or air-gapped installations, where internet access might not be desirable or possible.

However, I will see if we can add `include_plotlyjs` as a parameter that defaults to `False` so you have an option of setting this to `True` if you have access to the internet ✅

**Comment by SiddhantSadangi** [comment]
Hey @iirekm 👋 

We've released this feature with `neptune==1.13.0`. You can find the release notes here: https://github.com/neptune-ai/neptune-client/releases/tag/1.13.0

Can you try it out and let us know if it works? 🤗

**Comment by iirekm** [comment]
> Hey @iirekm 👋
> 
> We've released this feature with `neptune==1.13.0`. You can find the release notes here: https://github.com/neptune-ai/neptune-client/releases/tag/1.13.0
> 
> Can you try it out and let us know if it works? 🤗

It works, thanks!
